dispatcher.go

```diff
@@ -0,0 +1,56 @@
+package inflight
+
+import (
+	"sync"
+)
+
+type Dispatcher struct {
+	lock              sync.Mutex
+	queues            []*Queue
+	requestsexecuting int
```

Comment:
Why is this field here?  I see the definition and uses of a function to compute this value.

Reply:
removed